data/json/bionics.json

```diff
@@ -1126,7 +1126,7 @@
     "id": "bio_memory",
     "type": "bionic",
     "name": { "str": "Enhanced Memory Banks" },
-    "description": "Your memory has been enhanced with small quantum storage drives.  While active, you learn - both through reading and practice - more quickly.  Additionally, you forget skills you've learned slower.",
+    "description": "Your memory has been enhanced with small quantum storage drives.  While active, you learn - both through reading and practice - more quickly.",
```

Comment:
Could you separate this string change to get this in during string freeze? Or even better would be to add the try-catch block as a separate PR